packages/react-app/src/components/AddDelegateModal.js

```diff
@@ -49,13 +50,14 @@ export default function AddDelegateModal({
     
 
     return (
-        <Overlay>
+        <ThemeProvider theme={themes}>
+            <Overlay>
             <AliasModal ref={modalRef} background={themes}>
                 <Item align="flex-start">
                     <H2 textTransform="uppercase" spacing="0.1em">
-                    <Span weight="200">Add </Span><Span bg="#674c9f" color="#fff" weight="600" padding="0px 8px">Delegate</Span>
+                    <Span weight="200"  weight="200" color={themes.scheme=="dark" ? "white" : "black"}>Add </Span><Span bg="#674c9f" color="#fff" weight="600" padding="0px 8px">Delegate</Span>
```

Comment:
Use appropriate color from theme object only. For reference, see AliasVerificationModal